pkg/local_workflows/data_transformation_workflow_test.go

```diff
@@ -192,12 +192,12 @@ func Test_DataTransformation_with_Sarif_and_SummaryData(t *testing.T) {
 	assert.IsType(t, local_models.LocalFinding{}, localFinding)
 	assert.Len(t, localFinding.Findings, 278)
 
+	totalHighFindings := int(localFinding.Summary.Counts.CountBy.Severity["high"])
```

Comment:
nitpick: consider including tests for `Counts.Count` and `Counts.CountBySuppressed`